server/domains/entities/inquiry/inquiry.go

```diff
@@ -1,48 +1,36 @@
 package inquiry
 
-import (
-	"github.com/h3poteto/fascia/server/infrastructures/inquiry"
-)
-
 // Inquiry is a entity for inquiry.
 type Inquiry struct {
 	ID             int64
 	Email          string
 	Name           string
 	Message        string
-	infrastructure *inquiry.Inquiry
+	infrastructure InquiryRepository
+}
+
+type InquiryRepository interface {
```

Comment:
__[Golint]__ exported type InquiryRepository should have comment or be unexported

<sub>posted by Sider</sub>